scripts/fix_package_naming.py

```
@@ -0,0 +1,109 @@
+                    print("\t[no-change] Original package name: "+orig_pkg+ "  ->  "+canonical)
+                    new_line = line
+                else:
+                    print("\t[ changed ] Original package name: "+orig_pkg+ "  ->  "+canonical)
+                    new_line = line.replace(orig_pkg, canonical, 1)            
+                new_lines.append(new_line)
+            else:
+                new_lines.append(line)
+        else:
+            new_lines.append(line)
+
+    with open(pipfile_path, 'w', encoding='utf-8') as f:
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)